app/views/verify/finance/new.html.slim

```diff
@@ -1,29 +1,34 @@
 - title t('idv.titles.session.finance')
 
 h1.h3.my0 = t('idv.titles.session.finance')
-p.mt-tiny.mb0 = t('idv.messages.finance.intro_ccn')
+p.mt-tiny.mb0
+  = t('idv.messages.finance.intro_ccn_html',
+      intro_help_link: link_to(t('idv.messages.finance.intro_ccn_help'), help_path))
+
 = simple_form_for(idv_finance_form, url: verify_finance_path,
     html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
   = hidden_field_tag 'idv_finance_form[finance_type]', :ccn
-  .clearfix.mxn1
+
+  .clearfix.mxn1.px1
     - Idv::FinanceForm.ccn_inputs.each do |key, label, html_options|
-      .px1
+      div
         = f.label key, label: label, class: 'block bold'
         p.mb1.italic = t('idv.messages.finance.hint')
```

Comment:
We should connect this hint to the input with an `aria` tag.  I did something similar here: https://github.com/18F/identity-idp/blob/master/app/views/verify/sessions/new.html.slim#L47-L49